src/main/java/com/debatetimer/service/poll/PollService.java

```diff
@@ -0,0 +1,47 @@
+package com.debatetimer.service.poll;
+
+import com.debatetimer.domain.customize.CustomizeTable;
+import com.debatetimer.domain.member.Member;
+import com.debatetimer.domain.poll.Poll;
+import com.debatetimer.domain.poll.PollStatus;
+import com.debatetimer.domain.poll.VoteInfo;
+import com.debatetimer.domainrepository.poll.CustomizeTableDomainRepository;
+import com.debatetimer.domainrepository.poll.PollDomainRepository;
+import com.debatetimer.domainrepository.poll.VoteDomainRepository;
+import com.debatetimer.dto.poll.response.PollCreateResponse;
+import com.debatetimer.dto.poll.response.PollInfoResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+public class PollService {
+
+    private final CustomizeTableDomainRepository customizeTableDomainRepository;
+    private final PollDomainRepository pollDomainRepository;
+    private final VoteDomainRepository voteDomainRepository;
+
+    @Transactional
+    public PollCreateResponse create(long tableId, Member member) {
+        CustomizeTable table = customizeTableDomainRepository.getByIdAndMember(tableId, member);
+        Poll poll = new Poll(null, table.getId(), member.getId(), PollStatus.PROGRESS,
+                table.getProsTeamName(), table.getConsTeamName(), table.getAgenda());
+        Poll savedPoll = pollDomainRepository.create(poll);
+        return new PollCreateResponse(savedPoll);
+    }
```

Comment:
- id 값이 null 인 경우가 있을거면, 해당 요소를 제외한 생성자가 있으면 좋겠어요!
- id 값이 null 인 경우는 처음 투표를 시작하는 거니까 PollStatus.PROGRESS 가 기본값이면 좋겠어요!

```suggestion
    @Transactional
    public PollCreateResponse create(long tableId, Member member) {
        CustomizeTable table = customizeTableDomainRepository.getByIdAndMember(tableId, member);
        Poll poll = new Poll(table.getId(), member.getId(), table.getProsTeamName(), table.getConsTeamName(), table.getAgenda());
        Poll savedPoll = pollDomainRepository.create(poll);
        return new PollCreateResponse(savedPoll);
    }
```

Reply:
✅  반영 완료